core/chains/evm/logpoller/log_poller.go

```diff
@@ -566,35 +610,13 @@ func (lp *logPoller) findBlockAfterLCA(ctx context.Context, current *types.Heade
 	return nil, errors.New("Reorg greater than finality depth")
 }
 
-func (lp *logPoller) assertInFilter(eventSigs []common.Hash, addresses []common.Address) error {
```

Comment:
removing this now that we have Unregister, its conceivable clients may still want access to older captured logs after unregistering